lib/server/createHttpServer.ts

```diff
@@ -10,9 +10,11 @@ import runFrameStandaloneBundleContent from "@tscircuit/runframe/standalone" wit
 // @ts-ignore
 import winterspecBundle from "@tscircuit/file-server/dist/bundle.js"
 import { getIndex } from "../site/getIndex"
-import kleur from "kleur"
 
-export const createHttpServer = async (port = 3020) => {
+export const createHttpServer = async (
+  port = 3020,
+  mainComponentPath?: string,
```

Comment:
good time to convert to named arguments, `port` was obvious but the path is not obvious at all